### PR TITLE
Setup Linters

### PR DESCRIPTION
**Pull request by JavierAybar**
Activities in this branch:

- [ ] Add HTML file.
- [ ] Add CSS file.
- [ ] Add test.md file to be ignored. 
- [ ] Add linters config.

### CODE REVIEW

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @JavierAybar          , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descri[ptive commit messages👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️

- [x] kindly consider removing the micro verse logo or replacing it with yours if you've got one to personalize your readme as it's yours not microverse's
 

- [x] kindly consider customizing your readme for each and every single section to make it related to your current project  while doing that consider adding full information on how someone can interact with your project in  your getting started section(if it's a command to run add it ,....)  that will make your readme professional check on the images below and fix that
Screen Shot 2023-05-30 at 18 14 08

- [x] after that update your author's section to contain you only (i mean your social media links as one author) as you are the only participant in this project while doing so make sure you add working links to improve user experience to the people who might want to reach you out through those links

- [x] Last but not least kindly consider removing the note under your license section as it was given to you as a guide to help you so removing it will make your readme look professional 😀


_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -283,6 +199,5 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (divinecharlotte) [inline README.md:179]: - [x] While removing the note as requested previously you forgot to add a working link to your MIT license kindly consider fixing that to make your readme interactive 
- [x] As requested in the previous review kindly consider updating each section circled  👇 below by giving information related to it and your current project to make your readme more descriptive and professional
Screen Shot 2023-05-30 at 19 59 21

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @JavierAybar       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descri[ptive commit messages👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -60,174 +53,96 @@ After you're finished please remove all the comments and instructions!
...
+- **[It shows the proper GitHub Flow]**
+- **[It is easy to navigate]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
```
Comment (TadesseDev) [inline README.md:112]: - [Optional] Please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed.

**Review by TadesseDev — APPROVED** [review]
### Hi @JavierAybar 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (JavierAybar) [reply README.md:112]: Hi, Tadesse Thanks for everything, I'll keep it in mind for future projects, thank you very much!
Is it necessary to delete the branch feature after the merge?

**Pull request merged**